opts/ip_test.go

```diff
@@ -0,0 +1,54 @@
+package opts
+
+import (
+	"net"
+	"testing"
+)
+
+func TestIpOptString(t *testing.T) {
```

Comment:
Yes it is covered by #13879, I've just rebased against and moved it into this file ; so no if anybody *stole* it, it's me :sweat_smile:.